.gitattributes

```diff
@@ -8,3 +8,5 @@
 /.travis.yml        export-ignore
 /.scrutinizer.yml   export-ignore
 /appveyor.yml       export-ignore
+/scenarios/symfony2/tests export-ignore
+/scenarios/symfony4/tests export-ignore
```

Comment:
This directory should be `.scenarios.lock`, not `scenarios`, right? Shouldn't we just exclude all of the scenarios from export?